nemo_skills/dataset/audiobench/prepare.py

```diff
@@ -0,0 +1,546 @@
+# Copyright (c) 2025, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""AudioBench Dataset Preparation for nemo-skills
+
+This script prepares AudioBench datasets for evaluation with nemo-skills.
+AudioBench is a comprehensive benchmark for evaluating speech and audio models
+across multiple tasks including ASR, translation, speech QA, and more.
+
+Usage:
+    python -m nemo_skills.dataset.audiobench.prepare --split test
+    python -m nemo_skills.dataset.audiobench.prepare --datasets librispeech_test_clean earnings21_test
+    python -m nemo_skills.dataset.audiobench.prepare --category nonjudge
+"""
+
+import argparse
+import json
+import os
+import shutil
+import subprocess
+import sys
+from pathlib import Path
+from typing import Dict, List
+
+import numpy as np
+import soundfile as sf
+from tqdm import tqdm
+
+# AudioBench datasets categorized by evaluation type
+JUDGE_DATASETS = [
```

Comment:
roughly how big is the audiobench datasets? I need to understand if this data will be too much for the CI or not.

Reply:
yes, it will be to much for any CI/CD. even 1 sample could be a lot, as it contain longform audio